### DISCUSSION TITLE
Share photos to friends with immich

### DISCUSSION THREAD

**Discussion by KasiBurnsRed** [immich-app/immich#12708] [Feature Request]
It would be nice, if there was a way to share photos or Albums or folders to friends or family which also have immich. So that the photos are shown in their timeline or in an album or something else. Maybe it's also possible also to import shared web albums from other self hosted photo softwares or cloud solutions from friends. That would be a good way to exchange photos easy in good quality other then WhatsApp&co.

**Comment by bo0tzz** [comment]
#728
